1-mandatory/1-writers.js

```diff
@@ -51,8 +51,10 @@ Exercise 1:
   "Hi, my name is {firstName} {lastName}. I am {age} years old, and work as a {occupation}."
 */
 function logAllWriters() {
-  // write your code to log all writers here
-}
+   for(let writer of writers){
+   console.log(`Hi, my name is ${writer.firstName} ${writer.lastName}. I am ${writer.age} years old, and work as a ${writer.occupation}.`);
+  }
+};
```

Comment:
Comment: Well done. 

Consider: Here is another way to do it. function logAllWriters() {
  // write your code to log all writers here
  for (let i = 0; i < writers.length; i++) {
    const writer = writers[i];
    console.log(`Hi, my name is ${writer.firstName} ${writer.lastName}. I am ${writer.age} years old, and work as a ${writer.occupation}.`);
  }
}